ui-particles-angular/projects/ui-particles-angular/src/scss/mat-override/mat-button.scss

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2021 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@mixin mat-button() {
+  .mat-button.mat-button-base,
+  .mat-icon-button.mat-button-base,
```

Comment:
border-radius is wanted for icon-button  ? 
![image](https://user-images.githubusercontent.com/4974420/158145946-bdeb93e8-0831-4d4d-a428-69efb96c1d4a.png)
In figma we see it in circle mode 
but here the click action does not make the circle mode as before. 

Probably too much detail 😁

Reply:
You're right ! I just pushed a new version where I add the brightness but don't change the radius for the icon buttons.